pkg/api/grpc/metadata/metadata.go

```diff
@@ -31,7 +31,7 @@ type ctxKey struct{}
 func FromIncomingContext(ctx context.Context) (MD, bool) {
 	md, ok := ctx.Value(ctxKey{}).(MD)
 	if !ok {
-		return nil, false
```

Comment:
Did you check where this func is being called? I don't think we should be changing a return of type `bool` to being an `error`. I don't know if the caller will expect that and it might cause other issues. 

The idea behind the error standardization for the Metadata API was to update `err` variables that are already in our code base to be our standardized error with more relevant error details.

Reply:
> Did you check where this func is being called? I don't think we should be changing a return of type `bool` to being an `error`. I don't know if the caller will expect that and it might cause other issues. 
> 
> The idea behind the error standardization for the Metadata API was to update `err` variables that are already in our code base to be our standardized error with more relevant error details. 

Thanks for suggesting I will look in these